website/.spelling

```diff
@@ -1102,6 +1102,7 @@ P3M
 PT12H
 STRING_ARRAY
 String.format
+UNNEST
```

Comment:
had to add it because some checks were failing because of it https://app.travis-ci.com/github/apache/druid/jobs/530470206

Reply:
Hmm, this is strange because that word should have been added in another PR. Maybe it was because some necessary change was not in master yet when the CI ran last time. Can we revert this change and see what Travis says now?